README.md

```diff
@@ -134,6 +135,16 @@ If the focus trap has not been activated or has not been paused, nothing happens
 
 Returns the `focusTrap`.
 
+### focusTrap.updateContainerElements()
+
+Update the element(s) that are used as containers for the focus trap.
+
+When you call the function `createFocusTrap`, you pass in an element (or selector), or an array of elements (or selectors) to keep the focus within.  This method simply allows you to update which elements to keep the focus within.
+
+A use case for this is found in focus-trap-react, where React `ref`'s may not be initialized yet, but when they are you want to have them be a container element.
+
+Returns the `focusTrap`.
```

Comment:
Marking this here so I don't forget as something to remove depending on your thoughts on my comment in `index.d.ts`...